test/integration/aaa_download_only_test.go

```diff
@@ -83,7 +84,7 @@ func TestDownloadOnly(t *testing.T) {
 
 				// checking binaries downloaded (kubelet,kubeadm)
 				for _, bin := range constants.KubernetesReleaseBinaries {
-					fp := filepath.Join(localpath.MiniPath(), "cache", v, bin)
+					fp := filepath.Join(localpath.MiniPath(), "cache", runtime.GOOS, v, bin)
```

Comment:
the test should check for linux for kubeadm,kubelet...always.
since those are run always inside linux

Reply:
done